test/kubemark/start-kubemark-master.sh

```diff
@@ -50,4 +48,8 @@ kubernetes/server/bin/kube-apiserver \
 	--secure-port=443 \
 	--basic-auth-file=/srv/kubernetes/basic_auth.csv &> /var/log/kube-apiserver.log &
 
+# kube-contoller-manager now needs running kube-api server to actually start
+sleep 5
```

Comment:
Can we make it in more robust way - e.g. wait until apiserver is responding to /healthz or sth like that?